develop/tutorials/articles/210-application-security/04-authentication-pipelines/01-auto-login.markdown

```diff
@@ -0,0 +1,90 @@
+# Auto Login
+
+While @product@ supports a wide variety of [authentication mechanisms](/discover/deployment/-/knowledge_base/7-0/liferay-portal-security), 
+you may use a home-grown system or some other product to authenticate users. If
+this is your situation, you can write an Auto Login component to support your
+authentication system. 
+
+Auto Login components can check if the request contains something (a cookie, an
+attribute) that can be associated with a user in any way. If the component can
+make that association, it can authenticate that user to @product@. 
+
+## Creating an Auto Login Component
+
+Create a [Declarative Services component](/develop/tutorials/-/knowledge_base/7-0/creating-modules-with-liferay-ide#creating-component-classes). 
```

Comment:
yup... I just brought the topic to the table because I did not know whether it had been addressed elsewhere. Since it is a topic that is recurrent I am totally fine with what you have done talking about it elsewhere and don't elaborating each time. Maybe a link to the explanation on a side note could be fine, if possible.